.pre-commit-config.yaml

```diff
@@ -34,9 +34,9 @@ repos:
   rev: v1.3.0
   hooks:
   - id: mypy
-    name: mypy 3.7 on cibuildwheel/
+    name: mypy 3.8 on cibuildwheel/
     exclude: ^cibuildwheel/resources/.*py$
-    args: ["--python-version=3.7"]
+    args: ["--python-version=3.8"]
     additional_dependencies: &mypy-dependencies
```

Comment:
FYI, the markdown-it-py cap below can be removed, it was there to support Python 3.7.

Reply:
I'd rather be explicit here if it's OK.

Reply:
I mean “markdown-it-py<3” can be removed. I can’t comment on the correct line.

Reply:
ok, I misunderstood completely.
will do